lms/services/youtube.py

```diff
@@ -41,16 +43,15 @@ def video_info(self, video_id: str) -> dict:
         """
 
         # Endpoint docs: https://developers.google.com/youtube/v3/docs/videos/list
-        resp = self._http.get(
+        json_resp: dict = self._http.get(
```

Comment:
I think this might be the first time I've seen a Python type annotation used on a local variable in Hypothesis code

Reply:
Haha. Well, I guess it's not really needed here, as it can be inferred.

I think this comes from my TypeScript mental model, and trying to got the `"I want it to be this type. If it isn't, please warn me"` way, instead of `"I let you tell me what the type is, and will find out later if it's wrong"`.